superbench/benchmarks/micro_benchmarks/cudnn_function.py

```diff
@@ -440,14 +442,23 @@ def _process_raw_result(self, cmd_idx, raw_output):
                     self._curr_run_index, cmd_idx, self._name, raw_output, str(e)
                 )
             )
-            return False
+            error = True
         if error:
-            logger.error(
-                'Error in running cudnn test - round: {}, index of cmd: {}, benchmark: {}, raw data: {}'.format(
-                    self._curr_run_index, cmd_idx, self._name, raw_output
+            if self._capability == 9.0 and 'CUDNN_STATUS_NOT_SUPPORTED' in raw_output:
+                logger.warning(
+                    'CUDNN_STATUS_NOT_SUPPORTED error in running cudnn test on Hopper GPU - round: {}, \
+                        index of cmd: {},  benchmark: {}, raw data: {}'.format(
+                        self._curr_run_index, cmd_idx, self._name, raw_output
+                    )
                 )
-            )
-            return False
+                self._result.add_result(metric.lower() + '_time', -1)
```

Comment:
We have return code, why do we need this?

Reply:
since the return code is to indicate the whole benchmark not each command/config, so I added -1 for failed config to keep the metrics